.env.example

```
@@ -105,15 +105,31 @@ NEXT_PUBLIC_ENABLE_FILECOIN_ARCHIVAL=
 NEXT_PUBLIC_LENS_GRAPHQL_URL=
+NEXT_PUBLIC_QR_INIT_URL=
+NEXT_PUBLIC_QR_POLL_URL=
 NEXT_PUBLIC_ORB_QR_INIT_URL=
 NEXT_PUBLIC_ORB_QR_POLL_URL=
+NEXT_PUBLIC_MEDIA_GATEWAY=
 NEXT_PUBLIC_ORB_MEDIA_GATEWAY=
+NEXT_PUBLIC_LENS_GATEWAY=
 NEXT_PUBLIC_ORB_LENS_GATEWAY=
+NEXT_PUBLIC_ARWEAVE_GATEWAY=
 NEXT_PUBLIC_ORB_ARWEAVE_GATEWAY=
+NEXT_PUBLIC_IPFS_GATEWAY=
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The NEXT_PUBLIC_IPFS_GATEWAY key is duplicated

(DuplicatedKey)